src/index.html

```diff
@@ -15,8 +15,57 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="header__logo_a"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="logo"
+          class="header__img"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__ul">
+          <li class="header__li">
+            <a
+              href="#"
+              class="is-active"
```

Comment:
Fix class everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```